buildtools/pom.xml

```diff
@@ -44,6 +44,7 @@
     <testng.version>7.3.0</testng.version>
     <commons-lang3.version>3.11</commons-lang3.version>
     <maven-shade-plugin.version>3.2.4</maven-shade-plugin.version>
+    <excludedGroups>myExcludedGroup</excludedGroups>
```

Comment:
what is this group about myExcludedGroup

Reply:
it's just to test within buildtools tests that the exclusions work as expected.

Reply:
I had some doubts so that's why I added it. I can remove it if it is unnecessary.

Reply:
I removed it. :)